client/blocks/product-purchase-features/index.jsx

```diff
@@ -1,13 +0,0 @@
-/** @format */
-
-/**
- * External dependencies
- */
-
-import React, { Component } from 'react';
-
-export default class ProductPurchaseFeatures extends Component {
-	render() {
-		return <div className="product-purchase-features">{ this.props.children }</div>;
```

Comment:
Confirming that this `className` doesn't seem to be used for styling anywhere :+1:

Reply:
yes! Thanks for double-checking!